util/git.go

```diff
@@ -14,3 +16,16 @@ func UpdateGitRepo(ctx context.Context, gitpath string) {
 	Debugf(ctx, "%s: run %s\n", gitpath, cmd)
 	CheckCmd(cmd.CombinedOutput())
 }
+
+func IsPathIgnoredByGit(ctx context.Context, gitpath string, filepath string) bool {
+	args := []string{"check-ignore", "--quiet", "--no-index", fmt.Sprintf("%s/", strings.TrimPrefix(filepath, gitpath))}
```

Comment:
filepath: `/home/runner/work/cdnjs/cdnjs/cdnjs/ajax/libs/adobe-sign-sdk/1.0.0`, gitpath: `/home/runner/work/cdnjs/cdnjs/cdnjs`

We want `ajax/libs/adobe-sign-sdk/1.0.0` or `./ajax/libs/adobe-sign-sdk/1.0.0`, not `/ajax/libs/adobe-sign-sdk/1.0.0`

Hmm:
```suggestion
	args := []string{"check-ignore", "--quiet", "--no-index", fmt.Sprintf(".%s/", strings.TrimPrefix(filepath, gitpath))}
```